src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,272 @@
+package subway;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.Sql.ExecutionPhase;
+
+@DisplayName("지하철 노선 관련 기능")
+@Sql(scripts = {"/truncate.sql"}, executionPhase = ExecutionPhase.BEFORE_TEST_METHOD)
+@SpringBootTest(webEnvironment = WebEnvironment.DEFINED_PORT)
+public class LineAcceptanceTest {
+    /**
+     * Given: 새로운 지하철 노선 정보를 입력하고,
+     * When: 관리자가 노선을 생성하면,
+     * Then: 해당 노선이 생성되고 노선 목록에 포함된다.
+     */
+    @DisplayName("지하철 노선을 생성한다")
+    @Test
+    void createLine() {
+        // Given
+        Long firstStationId = requestCreateStation("지하철역")
+                                        .jsonPath()
+                                        .getObject("id", Long.class);
+        Long secondStationId = requestCreateStation("새로운지하철")
+                                        .jsonPath()
+                                        .getObject("id", Long.class);
```

Comment:
```suggestion
        var 지하철역 = requestCreateStation("지하철역");
        var 새로운지하철 = requestCreateStation("새로운지하철");
```

requestCreateStation에서 id까지 반환하는 건 어떨까요? 추가로 지하철역, 새로운지하철보다 일상생활에서 친근하게 볼 수 있는 지하철역 이름으로 변경하면 더 좋을 것 같아요!